cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -30,11 +30,12 @@ spec:
         - -initial-cluster-token
         - skydns-etcd
       - name: kube2sky
-        image: gcr.io/google_containers/kube2sky:1.7
+        image: gcr.io/google_containers/kube2sky:1.8
         args:
         # command = "/kube2sky"
         - -domain={{ pillar['dns_domain'] }}
         - -kubecfg_file=/etc/dns_token/kubeconfig
+        - -kube_master_url=https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}
```

Comment:
This PR is not intended to address #8314. This is meant to make a kube2sky release that does not connect to the master insecurely by default.